src/funds/funds_meta.py

```diff
@@ -0,0 +1,291 @@
+        _15y_change = self.parse_csv_float(values_basic["15y_change"])
+        _20y_change = self.parse_csv_float(values_basic["20y_change"])
+        profit_projection = self.parse_csv_float(values_basic["profit_projection"])
+        profit_projection_date = self.parse_csv_date(values_basic["profit_projection_date"])
+
+        return FundMeta(
+                        id=self.create_id(fund_code=code),
+                        fund_code=code,
+                        fund_id=fund_id,
+                        color=fund_json_entry["color"],
+                        kiid=fund_json_entry.get("kiid", None),
+                        long_name=fund_json_entry["longName"],
+                        name=fund_json_entry["name"],
+                        risk=fund_json_entry["risk"],
+                        short_name=fund_json_entry["shortName"],
+                        subs_name=fund_json_entry.get("subsName", None),
+                        group=self.group_map[group["group"]],
+                        price_date=price_date,
+                        a_share_value=a_share_value,
+                        b_share_value=b_share_value,
+                        _1d_change=_1d_change,
+                        _1m_change=_1m_change,
+                        _1y_change=_1y_change,
+                        _3y_change=_3y_change,
+                        _5y_change=_5y_change,
+                        _10y_change=_10y_change,
+                        _15y_change=_15y_change,
+                        _20y_change=_20y_change,
+                        profit_projection=profit_projection,
+                        profit_projection_date=profit_projection_date
+                      )
+
+    def parse_csv_date(self, str: str) -> Optional[date]:
+        """Parses date from CSV value
+
+        Args:
+            str (str): CSV value
+
+        Returns:
+            Optional[date]: date
+        """
+        if "-" == str:
+            return None
+
+        return datetime.strptime(str, "%d.%m.%Y").date()
+
+    def parse_csv_float(self, str: str) -> Optional[float]:
+        """Parses float from CSV value
+
+        Args:
+            str (str): CSV value
+
+        Returns:
+            Optional[float]: float
+        """
+        if "-" == str:
+            return None
+
+        return float(str.replace(",", "."))
+
+    def get_fund_values_basic_for_fund_id(self, fund_id: str) -> dict[str, str]:
+        """Resolves CSV row from basic values basic CSV for given fund_id
+
+        Args:
+            fund_id (str): fund id
+
+        Returns:
+            dict[str, str]: CSV row data
+        """
+        return next((entry for entry in self.get_fund_values_basic() if fund_id == entry["fund_id"]), None)
+
+    def load_funds(self) -> dict:
+        """Loads fund JSON file
+
+        Returns:
+            dict: JSON object
+        """
+        with open(os.environ["FUND_JSON"]) as json_file:
```

Comment:
I don't understand this access from design point of view. If these files exist in test directory(tests/data), then they are supposed to be used for tests, but it seems that we are accessing to them from implementation. Maybe they should be in a "common" directory in src? Also we need to check if the "path" is found too.

Reply:
In tests we are using test data, in production there will be production data that will be mounted as an external file